Sources/MapboxDirections/Intersection.swift

```diff
@@ -243,10 +261,17 @@ extension Intersection: Codable {
         
         var lanes: [Lane]?
         if let approachLanes = approachLanes,
-            let usableApproachLanes = usableApproachLanes {
+            let usableApproachLanes = usableApproachLanes,
+            let preferredApproachLanes = preferredApproachLanes,
+            let usableLaneIndication = usableLaneIndication {
             lanes = approachLanes.map { Lane(indications: $0) }
+            // document assumptions and call them out in PR
```

Comment:
Is this comment still needed? Did it end up being the case that we can assume there won’t be any contradictory active lane indications in the Directions API response?

Reply:
I will follow up with the Directions team. Their example has contradictory indications, but the explanation of the `valid_indication` in the docs implies that there should not be contradiction.

Reply:
Confirmed with Directions team that there won't be any contradictory active lane indications in the Directions API response.